wmi.h

```diff
@@ -0,0 +1,9 @@
+#ifndef __WMI_H__
```

Comment:
In regards to file names, "Google C++ Style Guide" states:
"Filenames should be all lowercase and can include underscores (_) or dashes (-). Follow the convention that your project uses. If there is no consistent local pattern to follow, prefer "_"."

Additionally, it states:
"C++ files should end in .cc and header files should end in .h.".

We should also keep names of source files consistent with corresponding header file.
wmi.h is inconsistent with WMI.cpp, taskutil.h is inconsistent with TaskUtil.cpp.

It seems that source code of Controller follows those rules.